apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+	type Control,
+	type FieldPath,
+	type FieldValues,
+} from "react-hook-form";
+import ReactSelect, {
+	type GroupBase,
+	type OptionsOrGroups,
+} from "react-select";
+
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+import { useFormController } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type Properties<T extends FieldValues> = {
+	control: Control<T, null>;
+	isMulti?: boolean;
+	name: FieldPath<T>;
+	options: OptionsOrGroups<unknown, GroupBase<unknown>> | undefined;
+	placeholder?: string;
+	title?: string;
+};
+
+const Select = <T extends FieldValues>({
+	control,
+	isMulti = false,
+	name,
+	options,
+	placeholder,
+	title,
+}: Properties<T>): JSX.Element => {
+	const { field } = useFormController({
+		control,
+		name,
+	});
+
+	return (
+		<div className={styles["container"]}>
+			{title && <span className={styles["title"]}>{title}</span>}
```

Comment:
Label is required for selects, in case we need to hide it, we will still have it, but with visually-hidden class. But for now it's not needed
```suggestion
			<span className={styles["label"]}>{title}</span>
```